File: Backend/TravellifeChaser/Models/AirlinesSystem/Airline.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace TravellifeChaser.Models
{
    public class Airline
    {
        public int Id { get; set; }

        public string Name { get; set; }

        public int AddressId { get; set; }
        public Address Address { get; set; }

        public string PromotionalDescription { get; set; }

        public double AverageRating { get; set; }

        public int PricelistId { get; set; }
        public Pricelist Pricelist { get; set; }

        public ICollection<Flight> Flights { get; set; } = new List<Flight>(); // N : 1  Airlines - Flight

        public ICollection<AirlineAirport> BuisinessDestinations { get; set; } = new List<AirlineAirport>();

        //public ICollection<Ticket> QuickReservationTickets { get; set; }  Flight has tickets...
        //public int AdminAirlinesUserId { get; set; }
        public AdminAirlinesUser Administrator { get; set; } 
    }
}

File: AvioAndRentACarSystem/src/app/AirlineModel/airline.ts
import { PriceList } from './priceList';
import { Airport } from './airport';
import { Address } from './address';
import { AirlineAirport } from './airlineAirport';

export class Airline
{
    id: number;
    name: string;
    address: Address;
    promotionalDescription: string;
    pricelist: PriceList;
    averageRating: number;
    businessDestinations: Array<AirlineAirport>;

    constructor(name?: string, address?: Address, promotionalDescription?: string, priceList?: PriceList)
    {
        this.name = name;
        this.address = address;
        this.promotionalDescription = promotionalDescription;
        this.pricelist = priceList;
        this.businessDestinations = new Array<AirlineAirport>();
    }
}